frontend/src/views/v-authorship.vue

```diff
@@ -159,13 +158,14 @@
           .ignored-segment
             .ignore-text File is ignored.
         pre.hljs.file-content(v-else-if="file.wasCodeLoaded", v-show="file.active")
-          v-segment-collection(v-bind:segments="file.segments", v-bind:path="file.path")
+          template(v-for="segment in file.segments")
```

Comment:
Looks like this PR is reverting some changes made in #1750. Was this unintentionally committed?